.github/workflows/build.yml

```diff
@@ -125,6 +204,7 @@ jobs:
         run: yarn test-e2e
 
   lint:
+    needs: [setup]
```

Comment:
Depending on `setup` is enough, since it performs prefetch for `ubuntu`. It's not that symmetrical like when `prefetch` does for all 3 platforms, but it's more practical to prefetch node modules in `setup` for `ubuntu`.